### PR TITLE
feat(Hitbloq): new presence

### PR DESCRIPTION
**Pull request by joerkig**
## Description 

Added a presence for [Hitbloq](https://hitbloq.com)

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Discord_P1LdXePUEs
Discord_nvLEcQkH4Y
Discord_67Zv2l73E7
Discord_Qf0BXCaxYR
Discord_9jk9dzIHiF
Discord_6ZJPtDHzlu
Discord_XysCwoCzBe
JU9MZOsQaF
Discord_wtBSQRAk0d
Discord_JIu4vD42a7

</details>

### CODE REVIEW

File: websites/H/Hitbloq/presence.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+			}
+			presenceData.smallImageText = `${characteristic} ${difficulty}`;
+			break;
+		}
+		case "map_pools": {
+			presenceData.details = "Browsing Map Pools";
+			break;
+		}
+		case "about": {
+			presenceData.details = "Viewing About Us";
```
Comment (EncryptedDev) [inline websites/H/Hitbloq/presence.ts:140]:
```suggestion
			presenceData.details = "Viewing the about us page";
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/H/Hitbloq/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		"name": "joerkig",
+		"id": "205984221859151873"
+	},
+	"service": "Hitbloq",
+	"description": {
+		"en": "Hitbloq is a competitive Beat Saber service with endless map pools where anything is rankable."
+	},
+	"url": "hitbloq.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/BtSGm7r.png",
```
[the remark this answers is not in the document]
Comment (joerkig) [reply websites/H/Hitbloq/dist/metadata.json:13]: Can it not be bigger than 512x512?

File: websites/H/Hitbloq/presence.ts [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+		case "actions": {
+			presenceData.details = "Viewing Action Queue";
+			break;
+		}
+		case "add_user": {
+			presenceData.details = "Viewing Add User";
+			break;
+		}
+		case "": {
+			presenceData.details = "Viewing Homepage";
```
[the remark this answers is not in the document]
Comment (joerkig) [reply websites/H/Hitbloq/presence.ts]: I'll put a space there and change capitalization but I disagree with putting "the" there. There is no "the" anywhere in the general localization file src with 2 minor exceptions, "the" will only cause the line to be longer than they have to be and should not be required, and if the localization file is wrong it should probably be fixed. This same critique also applies to the other suggestions in this PR where "the" was added.

File: websites/H/Hitbloq/dist/metadata.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (EncryptedDev) [reply websites/H/Hitbloq/dist/metadata.json:13]: Yikes, misread the size.

File: websites/H/Hitbloq/presence.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (EncryptedDev) [reply websites/H/Hitbloq/presence.ts]: The use of "the" is still preferable here

[the remark this answers is not in the document]
Comment (joerkig) [reply websites/H/Hitbloq/presence.ts]: 😔

[the remark this answers is not in the document]
Comment (joerkig) [reply websites/H/Hitbloq/presence.ts]: Done 😞

**Review by EncryptedDev — APPROVED** [review]

File: websites/H/Hitbloq/presence.ts [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+			presenceData.details = "Adding user";
+			break;
+		}
+		case "": {
+			presenceData.details = "Viewing the home page";
+			break;
+		}
+	}
+
+	cover;
```
Comment (Bas950) [inline websites/H/Hitbloq/presence.ts:163]: just random cover?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/H/Hitbloq/presence.ts (hunk 4 above)
Comment (joerkig) [reply websites/H/Hitbloq/presence.ts:163]: Oof, left over dev thing

**Review by Bas950 — APPROVED** [review]

**Pull request merged**